File: OPENAPI-REST-API/swagger-client/micronaut/src/main/java/io/swagger/model/InlineResponse20019.java
package io.swagger.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.swagger.model.InlineResponse20019Ips;
import io.swagger.model.InlineResponse20019PackageCosts;
import io.swagger.model.InlineResponse20019ServiceTypes;
import io.swagger.v3.oas.annotations.media.Schema;
import java.util.ArrayList;
import java.util.List;
import io.micronaut.core.annotation.Introspected;
import io.micronaut.validation.Validated;
import javax.validation.Valid;
import javax.validation.constraints.*;

/**
 * InlineResponse20019
 */
@Validated
@Introspected

public class InlineResponse20019   {
  @JsonProperty("packageCosts")
  private InlineResponse20019PackageCosts packageCosts = null;

  @JsonProperty("serviceTypes")
  @Valid
  private List<InlineResponse20019ServiceTypes> serviceTypes = null;

  @JsonProperty("ips")
  @Valid
  private List<InlineResponse20019Ips> ips = null;

  public InlineResponse20019 packageCosts(InlineResponse20019PackageCosts packageCosts) {
    this.packageCosts = packageCosts;
    return this;
  }

  /**
   * Get packageCosts
   * @return packageCosts
  **/
  @Schema(description = "")
  @NotNull

  @Valid
  public InlineResponse20019PackageCosts getPackageCosts() {
    return packageCosts;
  }

  public void setPackageCosts(InlineResponse20019PackageCosts packageCosts) {
    this.packageCosts = packageCosts;
  }

  public InlineResponse20019 serviceTypes(List<InlineResponse20019ServiceTypes> serviceTypes) {
    this.serviceTypes = serviceTypes;
    return this;
  }

  public InlineResponse20019 addServiceTypesItem(InlineResponse20019ServiceTypes serviceTypesItem) {
    if (this.serviceTypes == null) {
      this.serviceTypes = new ArrayList<InlineResponse20019ServiceTypes>();
    }
    this.serviceTypes.add(serviceTypesItem);
    return this;
  }

  /**
   * Get serviceTypes
   * @return serviceTypes
  **/
  @Schema(description = "")
  @NotNull
  @Valid
  public List<InlineResponse20019ServiceTypes> getServiceTypes() {
    return serviceTypes;
  }

  public void setServiceTypes(List<InlineResponse20019ServiceTypes> serviceTypes) {
    this.serviceTypes = serviceTypes;
  }

  public InlineResponse20019 ips(List<InlineResponse20019Ips> ips) {
    this.ips = ips;
    return this;
  }

  public InlineResponse20019 addIpsItem(InlineResponse20019Ips ipsItem) {
    if (this.ips == null) {
      this.ips = new ArrayList<InlineResponse20019Ips>();
    }
    this.ips.add(ipsItem);
    return this;
  }

  /**
   * Get ips
   * @return ips
  **/
  @Schema(description = "")
  @NotNull
  @Valid
  public List<InlineResponse20019Ips> getIps() {
    return ips;
  }

  public void setIps(List<InlineResponse20019Ips> ips) {
    this.ips = ips;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    InlineResponse20019 inlineResponse20019 = (InlineResponse20019) o;
    return Objects.equals(this.packageCosts, inlineResponse20019.packageCosts) &&
        Objects.equals(this.serviceTypes, inlineResponse20019.serviceTypes) &&
        Objects.equals(this.ips, inlineResponse20019.ips);
  }

  @Override
  public int hashCode() {
    return Objects.hash(packageCosts, serviceTypes, ips);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class InlineResponse20019 {\n");
    
    sb.append("    packageCosts: ").append(toIndentedString(packageCosts)).append("\n");
    sb.append("    serviceTypes: ").append(toIndentedString(serviceTypes)).append("\n");
    sb.append("    ips: ").append(toIndentedString(ips)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/inline_response20019.dart
part of swagger.api;

class InlineResponse20019 {
  
  InlineResponse20019PackageCosts packageCosts = null;

  List<InlineResponse20019ServiceTypes> serviceTypes = [];

  List<InlineResponse20019Ips> ips = [];

  InlineResponse20019();

  @override
  String toString() {
    return 'InlineResponse20019[packageCosts=$packageCosts, serviceTypes=$serviceTypes, ips=$ips, ]';
  }

  InlineResponse20019.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    packageCosts = new InlineResponse20019PackageCosts.fromJson(json['packageCosts']);
    serviceTypes = InlineResponse20019ServiceTypes.listFromJson(json['serviceTypes']);
    ips = InlineResponse20019Ips.listFromJson(json['ips']);
  }

  Map<String, dynamic> toJson() {
    return {
      'packageCosts': packageCosts,
      'serviceTypes': serviceTypes,
      'ips': ips
     };
  }

  static List<InlineResponse20019> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineResponse20019>() : json.map((value) => new InlineResponse20019.fromJson(value)).toList();
  }

  static Map<String, InlineResponse20019> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, InlineResponse20019>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new InlineResponse20019.fromJson(value));
    }
    return map;
  }
}
